plugins/org.yakindu.sct.ui.editor/src/org/yakindu/sct/ui/editor/editor/figures/TransitionFigure.java

```diff
@@ -6,27 +6,29 @@
  * http://www.eclipse.org/legal/epl-v10.html
  * Contributors:
  * 	committers of YAKINDU - initial API and implementation
- * 
+ *
  */
 package org.yakindu.sct.ui.editor.editor.figures;
 
 import java.lang.reflect.Field;
 
+import org.eclipse.draw2d.Graphics;
 import org.eclipse.draw2d.PolygonDecoration;
 import org.eclipse.draw2d.RotatableDecoration;
 import org.eclipse.gmf.runtime.draw2d.ui.figures.PolylineConnectionEx;
 import org.eclipse.gmf.runtime.draw2d.ui.mapmode.IMapMode;
+import org.yakindu.sct.ui.editor.editor.themes.ThemeProvider;
 
 /**
- * 
+ *
  * @author muelder
- * 
+ *
  */
 public class TransitionFigure extends PolylineConnectionEx {
 
 	private final IMapMode mapMode;
 
-	protected static final int TOLERANCE = 4;
+	protected static final int TOLERANCE = 3;
```

Comment:
why that?

Reply:
I think that was a mistake, I will remove it.